### PR TITLE
Skip transparent background for cursor

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: src/renderer/renderScene.ts [hunk 1]
```diff
@@ -22,9 +22,13 @@ type HandlerRectanglesRet = keyof ReturnType<typeof handlerRectangles>;
 function colorsForClientId(clientId: string) {
   // Naive way of getting an integer out of the clientId
   const sum = clientId.split("").reduce((a, str) => a + str.charCodeAt(0), 0);
+
+  // Skip transparent background.
+  const backgrounds = colors.elementBackground.splice(1);
+  const strokes = colors.elementStroke.splice(1);
```
Comment (j-f1) [inline src/renderer/renderScene.ts:28]: Shouldn’t this be `slice`? `splice` is the one that mutates.

Comment (lipis) [reply src/renderer/renderScene.ts:28]: oups...

**Comment by vjeux** [comment]
Fwiw, the stroke is black, so it would still be visible, just a black outline. We should still do it but it's not as broken as it may seem when reading the issue.

**Comment by lipis** [comment]
It's also used as stroke color for the selections in #1207 and neither black or transparent are good.. so it's a good change!

**Pull request merged**